aliveim.go

```diff
@@ -56,7 +62,11 @@ func handleAlivePost(rw http.ResponseWriter, request *http.Request) {
 	} else {
 		timer := time.NewTimer(time.Millisecond * time.Duration(aliverequest.Timeout))
 		deviceTimer := DeviceTimer{aliverequest.DeviceID, timer}
+
+		mutex.Lock()
 		timersMap[aliverequest.DeviceID] = deviceTimer
+		mutex.Unlock()
+
 		go deviceTimer.startTimer()
```

Comment:
@alessio sure, but we are locking ```timersMap``` not ```deviceTimer```. Anyway, not a big issue for me, I can move the ```.Unlock()``` right after that.